core/genesis.go

```diff
@@ -436,7 +438,7 @@ func GenesisToBlock(g *genesisT.Genesis, db ethdb.Database) *types.Block {
 	}
 	var withdrawals []*types.Withdrawal
 	if conf := g.Config; conf != nil {
-		isShangai := conf.IsEnabledByTime(g.Config.GetEIP4895TransitionTime, &g.Timestamp)
+		isShangai := conf.IsEnabledByTime(g.Config.GetEIP4895TransitionTime, &g.Timestamp) || g.Config.IsEnabled(g.Config.GetEIP4895Transition, new(big.Int).SetUint64(g.Number))
```

Comment:
Btw, I found out recently, that on the last release merge for some reason I used vars named as `isShangai` instead of `isEip4895`. I am thinking to revisit them and change them for consistency. What do you think guys?

Reply:
I will handle those in a different PR. Resolving this conversation now.